src/index.js

```diff
@@ -582,7 +582,7 @@ module.exports = async (request, response, config = {}, methods = {}) => {
 		try {
 			stats = await handlers.stat(absolutePath);
 		} catch (err) {
-			if (err.code !== 'ENOENT') {
+			if (err.code !== 'ENOENT' && err.code !== 'ENOTDIR') {
```

Comment:
Wouldn't this cause problems with rewrites? When there directory might not exist but a rewrite for it does?

Reply:
Not sure I follow, could you provide an example of what you mean?

Reply:
Thought there would be problems with such config where `static/b` doesn't exist

``` js
await handler(req, res, {
  'rewrites': [
    {source: 'static/b/**', destination: '/static/index.html'}
  ],
});
```

But can confirm, no such problem, my bad.